install_config/topics/sssd_for_ldap_failover_overview.adoc

```diff
@@ -0,0 +1,31 @@
+////
+sssd for ldap failover and extended attributes overview
+
+Module included in the following assemblies:
+
+* install_config/sssd_for_ldap_failover.adoc
+////
+
+[id='sssd-for-ldap-overview-{context}']
+
+{product-title} provides an
+xref:../configuring_authentication.adoc#LDAPPasswordIdentityProvider[authentication
+provider] for use with Lightweight Directory Access Protocol (LDAP) setups, but
+it can connect to only a single LDAP server. During {product-title} installation,
+you can configure the System Security
+Services Daemon (SSSD) for LDAP failover to ensure access to your cluster if one 
+LDAP server fails.
+
+The setup for this configuration is advanced and requires a separate
+authentication server, also called an *remote basic authentication server*, for
+{product-title} to communicate with. You configure this server
+to pass extra attributes, such as email addresses, to {product-title} so it can
+display them in the web console.
+
+This topic describes how to complete this set up on a dedicated physical or 
+virtual machine (VM), but you can also configure SSSD in containers.
+
+[IMPORTANT]
+====
+You must complete all sections of this topic.
```

Comment:
@kalexand-rh - Hey Kathryn, you may need to expand this note. In single page views on the portal, this may cause confusion (multi-page and d.o.c it is clear what it means).